test/generator-base-private.spec.js

```diff
@@ -158,17 +158,19 @@ export * from './entityFolderName/entityFileName.state';`;
     describe('generateTestEntityId', () => {
         describe('when called with int', () => {
             it('return 123', () => {
-                expect(BaseGenerator.generateTestEntityId('int')).to.equal(123);
+                expect(BaseGenerator.generateTestEntityId([{ field: { fieldType: 'Long' } }])).to.equal('123');
             });
         });
         describe('when called with String', () => {
             it("return '123'", () => {
-                expect(BaseGenerator.generateTestEntityId('String')).to.equal("'123'");
+                expect(BaseGenerator.generateTestEntityId([{ field: { fieldType: 'String' } }])).to.equal("'123'");
             });
         });
         describe('when called with UUID', () => {
             it("return '9fec3727-3421-4967-b213-ba36557ca194'", () => {
-                expect(BaseGenerator.generateTestEntityId('UUID')).to.equal("'9fec3727-3421-4967-b213-ba36557ca194'");
+                expect(BaseGenerator.generateTestEntityId([{ field: { fieldType: 'UUID' } }])).to.equal(
```

Comment:
Revert too.

Reply:
I would at least keep one like this to test my code too